src/http-api/resources/config.js

```diff
@@ -124,3 +127,61 @@ exports.show = (request, reply) => {
     return reply(config)
   })
 }
+
+exports.replace = {
+  // pre request handler that parses the args and returns `config` which is assigned to `request.pre.args`
+  parseArgs: (request, reply) => {
```

Comment:
was trying to keep consistency with the `getOrSet` endpoint, as the config file is still an argument :)